README.md

```diff
@@ -87,6 +87,7 @@ The model learns a mapping from (1) to both (2) and (3); in other words, from th
 
 #### Tutorial
 
+NOTE: Tutorials are not compatible across different branches. Currently tutorials are only compatible with the master branch.
```

Comment:
is the tutorial only compatible with `master` branch or with the branch it belong to? e.g. the changes in this PR update the tutorials to use the new args, but those will only be available in `dev-v0.3.0` branch right?

Reply:
It's only compatible with `master` branch, because we have a copy of configs uploaded on S3 which are compatible with master branch. Even though the tutorials.md files here are updated, they aren't yet compatible with uploaded data. 

We should begin some sort of versioning of the tutorials data and then we can update the wget links to fetch proper versions. I could create a folder dev-0.3.0 on S3, but I wanted to figure out a better startegy and then have a PR that updates it.

Reply:
I like the tutorial versioning idea. Let's figure that one out asap. The issue with the changes in this PR is that the tutorial says it only works with master branch, but then it uses options that are newly added to v0.3.0. I think this can be confusing to the user. With versioning that will absolutely get better